noir-projects/noir-contracts/contracts/nft_contract/src/test/transfer_to_private.nr

```diff
@@ -20,31 +20,23 @@ unconstrained fn transfer_to_private_to_self() {
     utils::assert_owns_public_nft(env, nft_contract_address, AztecAddress::zero(), token_id);
 }
 
+/// External orchestration means that the calls to prepare and finalize are not done by the NFT contract. This flow
+/// will typically be used by a DEX.
 #[test]
-unconstrained fn transfer_to_private_to_a_different_account() {
+unconstrained fn transfer_to_private_external_orchestration() {
```

Comment:
Aren't we missing a non-orchestrated test?

Reply:
We do not, but it was obfuscated in the code. Clarified that in https://github.com/AztecProtocol/aztec-packages/pull/9150/commits/dc88b65143ecd9e73a28aa684cb51ebd1cbbbfc2